public/index.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <!-- CSS goes here -->
     <link rel="stylesheet" type="text/css" href="./main.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,600;1,700;1,800&display=swap" rel="stylesheet">
```

Comment:
We'll need to use a local version of the font, as we want to be able to use chorus when offline